aten/src/ATen/native/QuantizedLinear.cpp

```diff
@@ -158,6 +158,28 @@ Tensor fbgemm_linear_int8_weight(
   return output.view(out_sizes);
 }
 
+Tensor fbgemm_linear_int8_weight(
+    const Tensor& input,
+    const Tensor& weight,
+    const Tensor& packed,
+    const Tensor& col_offsets,
+    Scalar weight_scale,
+    Scalar weight_zero_point,
+    const Tensor& bias) {
+  TORCH_WARN(
+      "fbgemm_linear_int8_weight will be deprecated soon."
+      "Please use fbgemm_linear_int8_weight_fp32_activation instead.");
+
+  return fbgemm_linear_int8_weight_fp32_activation(
```

Comment:
If you are registering something with native_functions, you must be explicit which function you are calling here -- otherwise this will be ambiguous.

Reply:
Sorry I might misunderstand it. How can I make the function call more explicit? I thought you want me to use the anonymous namespace to solve this issue, but it doesn't work.

Reply:
Sorry about the confusion. The problem with anonymous namespace is that it hides it from external use, but within the same file, it is visible. That makes it ambiguous if the function should be used from "nativeFunctions.h" or from the current file. If you explicitly specify the namespace (say `at::...`), it will be less confusing fro the compiler.